varia/sass/blocks/cover/_editor.scss

```diff
@@ -12,16 +12,15 @@
 		color: #{map-deep-get($config-cover, "color", "foreground")};
 	}
 
-    .wp-block-cover__inner-container,
```

Comment:
The only meaningful change is removing this selector. Everything else is linting + the result of rebuilding the varia themes.